.github/workflows/windows.yaml

```diff
@@ -7,23 +7,25 @@ jobs:
   test:
     env:
       DOWNLOAD_CACHE: 'd:\downloadcache'
+      GOPATH: 'd:\a\_temp\go\work'
```

Comment:
I think what we're after is putting both the go build/test cache and mod cache in this directory. I think I'd rather we use `GOMODCACHE` instead of `GOPATH` here then (just in case anything else sneaks in under `GOPATH`).

Reply:
Alright, sounds good. Since this basically involves breaking the cache again, I'm going to go ahead and give a shot at making the setup-go@v4 path work; these are the two directories that setup-go@v4 caches, so it ought to be at least similar.